src/structures/user.ts

```diff
@@ -14,6 +14,8 @@ export class User {
   readonly avatar?: string;
   /** Whether the user is a bot. */
   readonly bot: boolean;
+  /** The user's locale */
+  readonly locale?: string;
```

Comment:
Pull Requests on other libraries suggest that the **user** locale is a sibling to it's **guild** counterpart on the interaction structure root.

Reply:
That they are both in the interaction payload?

Reply:
Mhm, but I wasn't able to find a sufficient example payload to fit the description. I may have looked at the structure incorrectly or missed a comment explaining this.

https://github.com/discord/discord-api-docs/pull/4265#discussion_r772609501 in the context of this review, it was questioned for it's position and and noted "bad practice" since the user object already has the locale property when fetch via OAuth2 identity scope.
> The note at the bottom also staying it would be present in all but `PingInteraction`.

Reply:
This seems like more of a discussion on whether or not the `locale` in the user object makes sense or is "bad practice". AFAIK no Discord dev has told anyone that the prop is in the wrong place. If you want to add on to the doc about the `locale` that's fine.